Baby Monitor/Source Files/Custom Views/BasePageViewController.swift

```diff
@@ -7,7 +7,17 @@ import UIKit
 
 class BasePageViewController: UIPageViewController {
 
-    init() {
+    let analytics: AnalyticsManager
+
+    let analyticsScreenType: AnalyticsScreenType
+
+    var className: String {
+        return String(describing: type(of: self))
+    }
+
+    init(analytics: AnalyticsManager, analyticsScreenType: AnalyticsScreenType) {
```

Comment:
"Write documentation for every method and property accessible outside the class."
Missing documentation in BasePageViewController.